### PR TITLE
navbar added

### PR DESCRIPTION
**Pull request by swstica**

### CODE REVIEW

File: nav/nav.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+}
+.nav-items{
+    padding: 0 3rem 0 3rem;
+    justify-content: space-between;
+}
+.nav-item{
+    text-align: center;
+    font-size: 1.15rem;
+    margin-top: 3px;
+}
```
Comment (Sahil25061999) [inline nav/nav.css]: class name can be improved.

File: nav/nav.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  }
+.nav-items-right{
+    justify-content: space-evenly;
+    gap: 2rem;
+    margin-right: -2.5rem;
+}
+
+.nav-item img{
+    padding: 0;
+    margin-top: -0.5rem;
```
Comment (Sahil25061999) [inline nav/nav.css]: You can remove this

**Review by Sahil25061999 — APPROVED** [review]
Looks good to me only require minor changes
A UI suggestion would be to keep the search icon towards the right.

File: nav/nav.css [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+.component-details{
+    padding: 1rem;
+}
+.nav-examples{
+    padding: 1rem;
+    margin-bottom: 2rem;
+}
+.navbar2{
+    padding: 0.5rem;
+    box-shadow: 2px 5px 10px rgba(194, 192, 192, 0.7);
```
Comment (Anurag021) [inline nav/nav.css:17]: one suggestion would be to change color from rgba to hex(#) like other colors used in your CL.

**Review by Anurag021 — APPROVED** [review]
Just one suggestion above, rest the code looks good to me.

**Pull request merged**